src/unix/notbsd/linux/mod.rs

```diff
@@ -728,6 +728,7 @@ extern {
 
 cfg_if! {
     if #[cfg(any(target_env = "musl",
+                 target_os = "fuchsia",
```

Comment:
So right now this is under linux, because that's where musl lives, and because the diff is really minimal, but I'm not sure that's the best long-term structure. I think it might be better for musl to be a toplevel under notbsd, and have both linux-musl and fuschsia go there, with additional fuchsia `#[cfg]` items there as needed.